New functionality for adding custom list of words

klarasindelarova opened the pull request:
- User can put his own text file as a parameter of the program.
- If the file is not in the correct format, user is informed and game is launched with default list.
- New instructions regarding the file added to readme.md.

klarasindelarova committed:
- User can put his own text file as a parameter of the program.
- If the file is not in the correct format, user is informed and game is launched with default list.
- New instructions regarding the file added to readme.md.

sindema8 commented on src/ks/hangedman/Game.java:
```diff
@@ -24,24 +30,40 @@ public void start() {
-                wordsFromFile.add(row);
+                String row = reader.nextLine().toLowerCase();
+                String[] singleWords = row.split("\\s+");
+                if (singleWords.length > 1) {
```
this could be simplified:

if (length == 1) or else

because in both conditions the code is the same now (when <1 or >1)

sindema8 commented on src/ks/hangedman/Game.java:
```diff
@@ -24,24 +30,40 @@ public void start() {
     }
 
+    public boolean isAlphabetic(String word) {
+        return word.matches("^[a-zA-Z]+$");
```
So this game works only in English?

klarasindelarova replied:
Yes. I've also put it into the readme file, so it's clear for the user.

sindema8 commented on src/ks/hangedman/Game.java:
```diff
@@ -24,24 +30,40 @@ public void start() {
+                        wordsFromFile.add(singleWords[0]);
+                    } else {
+                        wordsFromFile.clear();
+                        break;
```
before each break, you could inform user, where is the problem, so he can better fix the file. If the file is large, it might be hard to localize, what is the problem, why you will not allow to use the list

klarasindelarova replied:
When reading of the file fails, the user receives info about it and also the problematic row is printed for easier localization. When there is an empty line, the program doesn't print the row, however - it won't be useful.
Also, I've added the line, where the program prints, that reading was successful and how many words it read, if that's the case.

klarasindelarova committed:
- If the file is not in the correct format, user receives more detailed info.
- readme.md updated again (special characters, only english words)

klarasindelarova committed:
- If the file is not in the correct format, user receives even more detailed info - the row with the mistake is printed.
- If the file is correct, user receives confirmation together with total count of his words.

sindema8 reviewed (approved):
👍
